lib/application_insights/channel/asynchronous_sender.rb

```diff
@@ -53,15 +53,13 @@ def initialize(service_endpoint_uri = SERVICE_ENDPOINT_URI)
       def start
         @start_notification_processed = false
         # Maintain one working thread at one time
-        unless @work_thread
-          @lock_work_thread.synchronize do
-            unless @work_thread
-              local_send_interval = [@send_interval, 0.1].max
-              @send_remaining_time = [@send_time, local_send_interval].max
-              @work_thread = Thread.new { run }
-              @work_thread.abort_on_exception = false
-            end
-          end
+        return if @work_thread
```

Comment:
The original double check is needed. It's a very common pattern, check out one example at here: https://www.vojtechruzicka.com/singleton-pattern-pitfalls/

For the changes in this PR, consider this scenario:
1. The ```@work_thread``` is nil, two threads calls the start() function, they both pass line 56.
1. Then thread A enters the synchronize section, thread B is blocked, a new work thread is created.
1. Thread B is unblocked and another new work thread is created.